tests/login.test.js

```diff
@@ -0,0 +1,52 @@
+//import $ from 'webdriverio/build/commands/browser/$';
+import loginPage from '../pages/login.page';
+
+describe('OpenBank', () => {
+
+    /* it('Should show login page', async () => {
+
+      browser.url('login');
+	  
+	  await browser.pause(5000);
+    }); */
```

Comment:
Remove commented code. This is an instance of **dead code** (Code you are not using, or was using at some point, but not anymore). Usually the less lines you have to maintain, the better.